File: acceptance/go-test-wrapper/driver/driver.go
package driver

import (
	"github.com/sirockin/cucumber-screenplay-go/back-end/pkg/entities"
)

// TestDriver is our interface to the system under test
type TestDriver interface {
	CreateAccount(name string) error
	ClearAll()
	GetAccount(name string) (entities.Account, error)
	Authenticate(name string) error
	IsAuthenticated(name string) bool
	Activate(name string) error
	CreateProject(name string) error
	GetProjects(name string) ([]entities.Project, error)
}

File: acceptance/ts-cucumber-playwright-bdd/src/drivers/driver.ts
import { Account, Project } from '../client/models';

export interface TestDriver {
  createAccount(name: string): Promise<void>;
  clearAll(): void;
  getAccount(name: string): Promise<Account>;
  authenticate(name: string): Promise<void>;
  isAuthenticated(name: string): boolean;
  activate(name: string): Promise<void>;
  createProject(name: string): Promise<void>;
  getProjects(name: string): Promise<Project[]>;
}